judge/management/commands/generate_sitemap.py

```diff
@@ -10,7 +10,7 @@
 
 
 class Command(BaseCommand):
-    requires_system_checks = False
+    requires_system_checks = ()
```

Comment:
what does this change do?

Reply:
According to [docs](https://docs.djangoproject.com/en/3.2/howto/custom-management-commands/#django.core.management.BaseCommand.requires_system_checks), there was a change to make this a list of tags instead of a boolean in Django 3.2. We missed it when upgrading.